keylime/cloud_verifier_tornado.py

```diff
@@ -129,24 +129,24 @@ def _from_db_obj(agent_db_obj: VerfierMain) -> Dict[str, str]:
 
 def verifier_read_policy_from_cache(stored_agent: VerfierMain) -> str:
     if stored_agent.agent_id not in GLOBAL_POLICY_CACHE:
-        GLOBAL_POLICY_CACHE[stored_agent.agent_id] = {}
+        GLOBAL_POLICY_CACHE[str(stored_agent.agent_id)] = {}
```

Comment:
I see ... the database should return here a string. Is it the complain that it is detecting a Column?

Reply:
no, I think that this makes sense